orchestrator/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/percona/percona-docker
```

Comment:
Hi, I'm not a Docker expert, but what is the purpose of this file?
I could imagine it is for building `peer-list` against 0.3.8 modules.
I see that in the 1st stage we build `peer-list` which is then copied to the 2nd stage, but is this file somehow visible to the 1st stage?

Reply:
@kamil-holubicki @surbhat1595 suggested just concept but you are welcome to rework PR as we don't have goland expertise

Reply:
@EvgeniyPatlan I'm not saying it is wrong. Just asking when, how, and what for this go.mod file is used.

Reply:
It is used to fix CVE issues in golang modules

Reply:
Thanks @kamil-holubicki , @EvgeniyPatlan ! I will look into the changes that needs to be done in orchestrator repo and then create the docker orchestrator images.